packages/ingest/src/granule.ts

```diff
@@ -247,15 +319,26 @@ function generateMoveFileParams(sourceFiles, destinations) {
  * @param {string} destinations.filepath - file path/directory on the bucket for the destination
  * @returns {Promise<Array>} returns array of source files updated with new locations.
  */
-async function moveGranuleFiles(sourceFiles, destinations) {
+export async function moveGranuleFiles(
+  sourceFiles: {
+    name: string,
+    bucket: string,
+    key: string
+  }[],
+  destinations: {
+    regex: string,
+    bucket: string,
+    filepath: string
+  }[]
+): Promise<{ bucket: string, key: string, name?: string }[]> {
   const moveFileParams = generateMoveFileParams(sourceFiles, destinations);
 
-  const processedFiles = [];
+  const processedFiles: { bucket: string, key: string, name?: string }[] = [];
```

Comment:
Same here: can we move this type into a definition so it's shared by the function return type?

Reply:
https://github.com/nasa/cumulus/pull/1798/commits/08ceba70690ea61f11cecfef5c6a71d1e1c13c47#diff-8b192e80bd09d39c934ec14baef99981R338